content/docs/gke/deploy/deploy-cli.md

```diff
@@ -132,72 +170,77 @@ Follow these steps to deploy Kubeflow:
     gcloud container clusters get-credentials ${KFAPP} --zone ${ZONE} --project ${PROJECT}
     ```
 
-   Then see what's installed in the `kubeflow` namespace of your GKE cluster:
+    Then see what's installed in the `kubeflow` namespace of your GKE cluster:
 
     ```
     kubectl -n kubeflow get all
     ```
 
-1. Access the Kubeflow central dashboard at the following URI when it becomes
-  available:
+## Access the Kubeflow user interface (UI)
 
-    ```
-    https://<KFAPP>.endpoints.<project-id>.cloud.goog/
-    ```
-   * It can take 20 minutes for the URI to become available.
-     Kubeflow needs to provision a signed SSL certificate and register a DNS 
-     name.
-   * If you own/manage the domain or a subdomain with 
-     [Cloud DNS](https://cloud.google.com/dns/docs/)
-     then you can configure this process to be much faster.
-     See [kubeflow/kubeflow#731](https://github.com/kubeflow/kubeflow/issues/731).
+Follow these steps to access the Kubeflow central dashboard:
 
-1. We recommend that you check in the contents of your **${KFAPP}** directory
-  into source control.
+1. Enter the following URI into your browser address bar. It can take 20
+  minutes for the URI to become available:
 
-## Understanding the deployment process
+  ```
```

Comment:
Users can also run the following command to get the host

```
kubectl -n istio-system get ingress
NAME            HOSTS                                          ADDRESS         PORTS   AGE
envoy-ingress   jlewi-v07-002.endpoints.jlewi-dev.cloud.goog   34.102.232.34   80      5d13h
```

Here's a one liner to set an environment variable to the host

```
export HOST=$(kubectl -n istio-system get ingress envoy-ingress -o=jsonpath={.spec.rules[0].host})
```

Reply:
Nice. Added.